z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -224,447 +208,107 @@ $ openstack project list
+| `dns_forwarder` | \<upstream DNS ip addr\> |For nameserver where requests should be forwarded for resolution.<br>**required** when use bastion server
+| `allocation_pool_start` |\<ip range start\> |'x.x.x.x'
+| `allocation_pool_end` |\<ip range end\> |'x.x.x.x'
+| `os_bootstrap_ip` | \<bootstrap ip addr\> |'x.x.x.x'
+| `os_master_ip` | \<master ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x]'
+| `os_infra_ip` |\<infra ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x]'
 
-**NOTE**: Only the API will be up at this point. The OpenShift UI will run on the compute nodes.
 
-### Delete the Bootstrap Resources
+## Create Cluster
 
-```sh
-$ ansible-playbook -i inventory.yaml down-bootstrap.yaml
-```
+- `$ ansible-playbook -i inventory.yaml 01-preparation.yaml`
 
-The teardown playbook deletes the bootstrap port, server and floating IP address.
+> Skip to run bastion.yaml if you want to use your existing DNS/HAProxy or external DNS/HAProxy, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update it.
 
-If you haven't done so already, you should also disable the bootstrap Ignition URL.
+- `ansible-playbook -i inventory.yaml bastion.yaml`
+- `ansible-playbook -i inventory.yaml 02-create-cluster-control.yaml`
+- `ansible-playbook -i inventory.yaml 03-create-cluster-compute.yaml`
 
-### OpenShift Compute Nodes
+After above steps, you will get one ready OpenShift Platform Cluster on ICIC infrastructure.
 
-```sh
-$ ansible-playbook -i inventory.yaml configure-compute-nodes.yaml
-```
+## Something is wrong? clean up environment
 
-This process is similar to the masters, but the workers need to be approved before they're allowed to join the cluster.
+### Destroy Cluster
+Depending on different cases to choose suitable destroying.
++ You might get some generic errors during playbook `01-preparation.yaml` and `bastion.yaml`,please use:
 
-The workers need no ignition override.
+`ansible-playbook -i inventory.yaml 04-destroy.yaml`
 
-### Configure the storage backend of image registry 
++ You might get some ICIC create control errors during playbook `02-create-cluster-control.yaml`,please use:
 
-```sh
-$ ansible-playbook -i inventory.yaml configure-image-registry.yaml
-```
+`ansible-playbook -i inventory.yaml 04-destroy.yaml`
 
-The Image Registry Operator is not initially available for platforms that do not provide default storage. We set the image registry to an empty directory for non-production clusters.
++ You might get some ICIC create compute server error during playbook `02-create-cluster-compute.yaml`,please use:
 
-And the image registry will be exposed, this allows you to log in to the registry from outside the cluster using the route address, and to tag and push images using the route host.
+`ansible-playbook -i inventory.yaml destroy-computes.yaml`
 
-Log in with podman: 
+## Day2 Operation
```

Comment:
Before the Day2 Operation Section or After the Day2 Operation , we need add one section named Destroy Cluster with description. 
## Destory Cluster
### If you don't need the OpenShift Container Platform , you can destory it :
```sh
$ ansible-playbook -i inventory.yaml 04-destroy.yaml
```

Reply:
Hi XiaoMei, I already updated this section 6 days ago with this comment.
```
 z_infra_provisioning/cloud_infra_center/ocp_upi/README.md Outdated

Upon success, it will print the URL to the OpenShift Console (the web UI) as well as admin username and password to log in.
## Destroy Cluster
@zhengxiaomei123 zhengxiaomei123 6 days ago Collaborator

Remove the title "Destroy Cluster" here and add the "Destroy Cluster" section after the "Something is wrong? clean up environment" part.
@shhull shhull 6 days ago Author Contributor

## Something is wrong? clean up environment

### Destroy Cluster
```

The new look view:
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/74761704/132150332-c75d17cc-66e9-4d89-a44c-89e6350403bf.png">
- Create Cluster
- Something is wrong? clean up environment
   --Destroy Cluster